src/main/java/codesquad/springcafe/user/UserController.java

```diff
@@ -0,0 +1,46 @@
+package codesquad.springcafe.user;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class UserController {
```

Comment:
```suggestion
@Controller
@RequestMapping("/user")
public class UserController {
```

요런식으로 대표 URL? 을 지정해 줄 수 있어요.